src/patternfly/base/tokens/_tokens-dark.scss

```diff
@@ -0,0 +1,296 @@
+/**
+ * Do not edit directly
+ * Generated on Mon, 14 Aug 2023 18:11:22 GMT
+ */
+
+ //  stylelint-disable custom-property-pattern
```

Comment:
If you wanna add support for this pattern, here's what you'd do it

https://github.com/patternfly/patternfly/blob/47689a42e7557e2967ae3d3974ddaa7fcf251064/.stylelintrc#L42

I imagine you'd just do something like change `v5` to `(v5|t)-(global|[lcud])`